community/kernel/src/main/java/org/neo4j/kernel/api/DataRead.java

```diff
@@ -116,24 +112,25 @@ long nodeGetFromUniqueIndexSeek( IndexDescriptor index, Object value ) throws In
      */
     PrimitiveIntIterator nodeGetLabels( long nodeId ) throws EntityNotFoundException;
 
-    PrimitiveIntIterator nodeGetAllPropertiesKeys( long nodeId ) throws EntityNotFoundException;
+    PrimitiveIntIterator nodeGetPropertyKeys( long nodeId ) throws EntityNotFoundException;
+
+    PrimitiveIntIterator relationshipGetPropertyKeys( long relationshipId ) throws EntityNotFoundException;
 
-    PrimitiveIntIterator relationshipGetAllPropertiesKeys( long relationshipId ) throws EntityNotFoundException;
+    PrimitiveIntIterator graphGetPropertyKeys();
```

Comment:
In Neo4j there are nodes, relationships and a global graph object (hardly ever used). This is for that graph object.